fastlane/Fastfile

```diff
@@ -233,6 +234,37 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       dsym_path: "./WooCommerce.app.dSYM.zip",
     )
     sh("cd .. && rm WooCommerce.app.dSYM.zip")
+
+    if (options[:create_release])
+      archive_zip_path = File.dirname(Dir.pwd) + "/WooCommerce.xarchive.zip"
+      zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
+
+      version = ios_get_app_version()
+      create_release(repository:GHHELPER_REPO, 
+        version: version, 
+        release_notes_file_path:'../WordPress/Resources/release_notes.txt',
+        release_assets:"#{archive_zip_path}"
+      )
+
+      sh("rm #{archive_zip_path}") 
+    end
+  end
+
+  #####################################################################################
+  # build_release
+  # -----------------------------------------------------------------------------------
+  # This lane builds the app, create a GH release and upload it for external distribution  
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_release [skip_confirm:<skip confirm>]
+  #
+  # Example:
+  # bundle exec fastlane build_release 
+  # bundle exec fastlane build_release skip_confirm:true 
+  #####################################################################################
+  desc "Builds, create release and uploads for distribution"
+  lane :build_release do | options |
```

Comment:
WDYT about making this `build_and_upload_release` to make it consistent with the `build_and_upload_installable_build`? Just thinking it doesn't really represent the fact that it uploads as well. That lane name already exists, but IMHO it'd be very reasonable for this to replace it?

Reply:
I've followed the approach I'm proposing on WPiOS here: I removed this lane and I'll make the existing `build_and_upload_release` smarter.